forms-flow-web-root-config/docker-compose.yml

```diff
@@ -43,6 +43,8 @@ services:
       - REACT_APP_ENABLE_DASHBOARDS_MODULE=${ENABLE_DASHBOARDS_MODULE:-true}
       - REACT_APP_ENABLE_PROCESSES_MODULE=${ENABLE_PROCESSES_MODULE:-true}
       - REACT_APP_ENABLE_APPLICATIONS_MODULE=${ENABLE_APPLICATIONS_MODULE:-true}
+      - REACT_APP_DATE_FORMAT=${DATE_FORMAT}
```

Comment:
Add to sample.env , config template/ sample and add in. Change log too about the env

Reply:
ok